R/as_html.R

```diff
@@ -61,7 +62,8 @@ as_html <- function(x,
                     class_tr = NULL,
                     class_td = NULL,
                     class_th = NULL,
-                    link_label = NULL) {
+                    link_label = NULL,
+                    bold_main_title = FALSE) {
```

Comment:
I think we could make bold options `rtables::theme_docx_default` has `bold = c("header", "content_rows", "label_rows")`.  I think it would be nice also to add horizontal lines as default's console output does. What do you think?

Reply:
Unfortunately, we can't add the horizontal lines around the table header since those are automatically added in some viewer windows (e.g. the teal apps), and adding them in here will mess up the formatting there. I have added (and generalized) the `bold` argument.

Reply:
Disregard my comment on the horizontal line - I got it working :)